src/main/java/graphql/execution/nextgen/FetchedValueAnalysis.java

```diff
@@ -72,7 +72,7 @@ public FetchedValue getFetchedValue() {
         return fetchedValue;
     }
 
-    public FetchedValueAnalysis transfrom(Consumer<Builder> builderConsumer) {
+    public FetchedValueAnalysis transform(Consumer<Builder> builderConsumer) {
```

Comment:
as an internal class we can change this - :)

Reply:
well I'll propose this @Internal annotation in my team as well... it is very useful 😊